pyfstat/loggers.py

```diff
@@ -0,0 +1,89 @@
+"""
+Description of logging implementation in PyFstat.
+
+We currently support 3 modes of operation, depending on user input:
+
+    1. Default logging
+    2.
+    3.
+"""
+
+import logging
+import os
+import sys
+
+from pyfstat.helper_functions import get_version_string
+
+
+def _parse_log_level(log_level):
```

Comment:
Why do you need this one? Isn't it best to always set levels as `logging.INFO` and similar? And if you need to do so from a string, I guess you could just do `getattr(logging,string)`?